src/framework/audio/internal/synthesizers/fluidsynth/soundmapping.h

```diff
@@ -807,114 +807,129 @@ inline const midi::Programs& findPrograms(const mpe::PlaybackSetupData& setupDat
 
 inline const ArticulationMapping& articulationSounds(const mpe::PlaybackSetupData& setupData)
 {
-    static ArticulationMapping ELECTRIC_GUITAR = {
+    static const ArticulationMapping ELECTRIC_GUITAR = {
```

Comment:
@RomanPudashkin This is what I missed when I tried it. What are those ArticulationMappings?

Reply:
They are needed for Fluid/soundfonts (MS Basic)